modules/optimizing-mtu-networking.adoc

```diff
@@ -7,6 +7,12 @@
 
 There are two important maximum transmission units (MTUs): the network interface controller (NIC) MTU and the cluster network MTU.
 
-The NIC MTU is configured at the time of {product-title} installation, and you can also change the cluster's MTU as a Day 2 operation. See "Changing cluster network MTU" for more information. The MTU must be less than or equal to the maximum supported value of the NIC of your network. If you are optimizing for throughput, choose the largest possible value. If you are optimizing for lowest latency, choose a lower value.
+The NIC MTU is configured at the time of {product-title} installation, and you can also change the MTU of a cluster as a postinstallation task. See "Changing cluster network MTU" for more information. 
+
+For a cluster that uses the OVN-Kubernetes plugin or, the MTU must be less than `100` bytes to the maximum supported value of the NIC of your network. If you are optimizing for throughput, choose the largest possible value. If you are optimizing for lowest latency, choose a lower value.
```

Comment:
This might have either been a typo, or it needs completion. Consider fixing it accordingly
```suggestion
For a cluster that uses the OVN-Kubernetes plugin, the MTU must be less than `100` bytes to the maximum supported value of the NIC of your network. If you are optimizing for throughput, choose the largest possible value. If you are optimizing for lowest latency, choose a lower value.
```